Libraries/src/Amazon.Lambda.Annotations/Amazon.Lambda.Annotations.csproj

```diff
@@ -4,8 +4,10 @@
     <AssemblyVersion>1.4.0</AssemblyVersion>
     <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-  </PropertyGroup>	
-	
+    <AssemblyOriginatorKeyFile>..\..\..\buildtools\public.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
```

Comment:
The `SQSEventAttribute` defined in `Amazon.Lambda.Annotations` contains some internal methods to indicate whether the properties of this attribute were set or not. The `Amazon.Lambda.Annotation.SourceGenerator` needs to access these internal methods. Both these assemblies need to be strongly signed for the `InternalsVisibleTo` to take effect.